core/src/main/kotlin/no/strooped/view/screen/TaskScreen.kt

```diff
@@ -70,21 +66,25 @@ label.getStyle().background = new Image(new Texture(labelColor)).getDrawable();*
             }
         }
         var colorClicked = false
```

Comment:
Can we instead place this as a property on the TaskScreen class, so it is more clear that this is a state affecting the whole screen, and not just a utility variable inside show?

Reply:
Done